mozilla-release/mobile/android/base/java/org/mozilla/gecko/controlcenter/OverviewFragment.java

```diff
@@ -87,33 +88,78 @@ public String getTitle(Context context) {
         return context.getString(R.string.cc_title_overview);
     }
 
+    private <T> T getValueFromBundle(@NonNull GeckoBundle bundle, @NonNull String key, T defaultValue, Class<T> clazz) {
```

Comment:
Let's move this to a utility class. Because this stuff can also be used in the other two views of the control center